### PR TITLE
Verify that Cliffords fix Pauli group

### PR DESCRIPTION
**Pull request by viathor**
Follow-up to #3414. Sorry, couldn't resist.

### CODE REVIEW

File: cirq/experiments/qubit_characterizations_test.py [hunk 1]
```diff
@@ -72,6 +81,11 @@ def check_distinct(unitaries):
         assert any(
             cirq.allclose_up_to_global_phase(Uxy, Uxz) for Uxz in xz_unitaries)
 
+    # Check that each unitary fixes the Pauli group.
+    for u in xy_unitaries:
+        for p in PAULIS:
+            assert is_pauli(u @ p @ np.conjugate(np.transpose(u))), f'{u}'
```
Comment (balopat) [inline cirq/experiments/qubit_characterizations_test.py:87]: I.conj().T would be a but shorter

Comment (viathor) [reply cirq/experiments/qubit_characterizations_test.py]: Done.

File: cirq/experiments/qubit_characterizations_test.py [hunk 2]
```diff
@@ -72,6 +81,11 @@ def check_distinct(unitaries):
         assert any(
             cirq.allclose_up_to_global_phase(Uxy, Uxz) for Uxz in xz_unitaries)
 
+    # Check that each unitary fixes the Pauli group.
+    for u in xy_unitaries:
+        for p in PAULIS:
+            assert is_pauli(u @ p @ u.conj().T), f'{u}'
```
Comment (maffoo) [inline cirq/experiments/qubit_characterizations_test.py:87]: nit: instead of `f'{u}'` can just do `str(u)`

**Review by maffoo — APPROVED** [review]
LGTM

File: cirq/experiments/qubit_characterizations_test.py (hunk 2 above)
Comment (viathor) [reply cirq/experiments/qubit_characterizations_test.py]: Done.

**Pull request merged**